app/src/main/java/openfoodfacts/github/scrachx/openfood/views/product/ingredients/IngredientsProductFragment.java

```diff
@@ -290,18 +293,24 @@ public void refreshView(State state) {
         presenter.loadAdditives();
 
         if (isNotBlank(product.getImageIngredientsUrl())) {
-            addPhotoLabel.setVisibility(View.GONE);
 
-            // Load Image if isLowBatteryMode is false
-            if (!isLowBatteryMode) {
-                Picasso.with(getContext())
-                        .load(product.getImageIngredientsUrl())
-                        .into(mImageIngredients);
+            if (!Locale.getDefault().getLanguage().equals(product.getLang())) {
+                addPhotoLabel.setText("Take a photo of the ingredients in "+Locale.getDefault().getDisplayLanguage());
```

Comment:
This text needs to be in the strings.xml, so that it can be translated.

Reply:
Yes, I'll fix this.

Reply:
All requested changes have been made.